app/components/Messages/PositionProfile/index.jsx

```diff
@@ -4,6 +4,8 @@ import T from 'prop-types';
 
 import iconDictionary from 'utils/iconDictionary';
 
+import {useNavigate} from "react-router-dom"
+
```

Comment:
Just for consistent styling, we keep all package imports to the top of the file (near the import `react` and  import `prop-types`)